github_rules/github_repo_initial_access.py

```diff
@@ -0,0 +1,23 @@
+from panther_oss_helpers import get_string_set, put_string_set
+
+
+def rule(event):
+    if event.get("action").startswith("git.") and not event.get("repository_public", False):
```

Comment:
The default return value of `get` is `None`, which is falsey, so I don't think you need to specify `False` as the default value here.
```suggestion
    if event.get("action").startswith("git.") and not event.get("repository_public"):
```